.github/workflows/run-tests.yml

```
@@ -0,0 +1,65 @@
+name: ci testing
+
+
+on:
+  push:
+    branches: master
+  pull_request:
+
+
+jobs:
+
+  run_test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [
+          3.7,
+          3.8,
+          3.9,
+        ]
+        include:
+          - python-version: 3.7
+            tox-env: py37
+          - python-version: 3.8
+            tox-env: py38
+          - python-version: 3.9
+            tox-env: py39
+
+    steps:
+
+      - name: cancel previous workflows that are still running
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: checkout latest commit
+        uses: actions/checkout@v2
+
+      - name: setup python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+
+      - name: linting and code style
+        if: matrix.python-version == '3.7'
+        run: tox -vv -e lint
+
+      - name: tests and coverage
+        run: tox -vv -e ${{ matrix.tox-env }}
```

```suggestion
        run: tox -vv -e ${{ matrix.tox-env }} -- --color=yes
```
This should give us nice colors, no? Maybe I am wrong though.

---

Good idea! Done in 6f5cf1780e4f3a4313a77a07729a6a8c3bdce16f